eth-types/src/evm_types.rs

```diff
@@ -126,6 +126,8 @@ impl GasCost {
     pub const MEMORY_EXPANSION_QUAD_DENOMINATOR: Self = Self(512);
     /// Coefficient of linear part of memory expansion gas cost
     pub const MEMORY_EXPANSION_LINEAR_COEFF: Self = Self(3);
+    /// constant gas for logs op codes
+    pub const LOG_STATIC_GAS: Self = Self(375);
```

Comment:
```suggestion
    pub const LOG: Self = Self(375);
```

Reply:
update in [755a06d](https://github.com/appliedzkp/zkevm-circuits/pull/335/commits/755a06d429eb994510af696e5c89d0c96f086176) thanks